src/components/CardListItem.js

```diff
@@ -18,29 +18,40 @@ const renderCardContent = (item, horizontal) => {
   const cardContent = [];
 
   const sequenceMap = {
-    picture: () => (
-      <Image
-        source={{ uri: picture.url }}
-        style={stylesWithProps({ horizontal }).image}
-        containerStyle={(styles.imageContainer, !!imageStyle && imageStyle)}
-        borderRadius={imageBorderRadius}
-      />
-    ),
-    topTitle: () => (
-      <RegularText small style={[!!generalStyle && generalStyle, !!topTitleStyle && topTitleStyle]}>
-        {topTitle}
-      </RegularText>
-    ),
-    subtitle: () => (
-      <RegularText small style={[!!generalStyle && generalStyle, !!subtitleStyle && subtitleStyle]}>
-        {subtitle}
-      </RegularText>
-    ),
-    title: () => (
-      <BoldText style={[!!generalStyle && generalStyle, !!titleStyle && titleStyle]}>
-        {horizontal ? (title.length > 60 ? title.substring(0, 60) + '...' : title) : title}
-      </BoldText>
-    )
+    picture: () =>
+      !!picture &&
+      !!picture.url && (
```

Comment:
maybe shorter with one check and `?` doing both checks? `!!picture?.url`

Reply:
Yeah, that's a good idea. 👍

https://github.com/smart-village-solutions/smart-village-app-app/commit/7849be8e32f6d20bc2353e771d1b587af66e022d